gluten-substrait/src/main/scala/org/apache/gluten/config/GlutenConfig.scala

```diff
@@ -778,6 +780,14 @@ object GlutenConfig {
       .booleanConf
       .createWithDefault(true)
 
+  val GLUTEN_ANSI_FALLBACK_ENABLED =
+    buildConf("spark.gluten.sql.ansiFallback.enabled")
```

Comment:
This config does the same thing as an existing one.
I think it would be better to add a whitelist-like config to cover the functions that are already supported, and then fall back to the corresponding plan based on it.